pkg/helper/options/options.go

```diff
@@ -25,21 +25,27 @@ import (
 type DeleteOptions struct {
```

Comment:
As a follow up, tests for `options.go` would be great. These lines are obviously covered by tests from various commands, but it would be nice to have explicit tests for these.

Reply:
> but it would be nice to have explicit tests for these.
Yes, I'll try to add a test later! thanks :smile:

Reply:
Sounds good! Thanks and great work on this.